R/outputs.R

```diff
@@ -183,8 +183,17 @@ extract_indicators <- function(naomi_fit, naomi_mf, na.rm = FALSE) {
                   indicator_anc_est_t3
                 )
 
-  dplyr::select(out, names(naomi_mf$mf_out),
+  out <- dplyr::select(out, names(naomi_mf$mf_out),
                 calendar_quarter, indicator, mean, se, median, mode, lower, upper)
+
+  #  Add a class to indicate if uncertainty has been generated
+  if(!is.null(naomi_fit$sample)) {
+    class(out) <- c("mean", "tbl_df", "tbl","data.frame")
+  } else {
+    class(out) <- c("mode", "tbl_df", "tbl","data.frame")
+  }
```

Comment:
1. Need more descriptive / specific class names.
2. I would be inclined to put the class name on the output object, rather than the `indicators` table specifically. Is that feasible to do?  Also, let's chat with @r-ash if that is a sensible use of class.

Reply:
Have added more specific class names and left the class on the indicators table as discussed